web/admin/src/PresentationDriver.js

```diff
@@ -290,6 +290,7 @@ export default class PresentationDriver extends PureComponent {
     if (!scores) return []
     const { users, session } = this.props
     let prevScore = Number.MAX_SAFE_INTEGER
+    let prevTime = Number.MAX_SAFE_INTEGER
```

Comment:
You are sorting by score descending, then by time *ascending* so this should start out as something less than `0` (e.g. `-1`)